tests/specfiles/gdb-exploitable-empty-buildarch.spec

```diff
@@ -0,0 +1,112 @@
+%global plugin exploitable
+%global snapshot 20210920gitab64075
+%global gdb_data /usr/share/gdb/python/%{plugin}
+%global gdb_init /etc/gdbinit.d
+
+# The tarball is created like this:
+# git clone https://github.com/jfoote/exploitable.git
+# tar --exclude=.* -cvf exploitable.tar exploitable/*
+# gzip --best exploitable.tar
+# Then the snapshot global is updated to meet fedora guidelines
+
+Name: gdb-%{plugin}
+Version: 1.32
+Release: 20.%{snapshot}%{?dist}
+Summary: GDB extension for exploitability
+License: MIT AND MIT-CMU AND BSD-3-Clause
+URL: https://github.com/jfoote/exploitable
+Source0: %{plugin}.tar.gz
+Source1: %{name}.gdb
+Source2: README.fedora
+Patch1: exploitable-distutils.patch
+ExclusiveArch: x86_64 i686
+Requires: gdb
```

Comment:
It seems we are copy-pasting + slightly changing the same specfile under a different name; can we have a a different (real-life) specfile instead?